index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="mystyle.css">
+    <link rel="stylesheet" href="css/all.min.css">
+    <title>Portfolio</title>
+</head>
+<body>
+
+<header>
+    <nav>
+    <a href="https://www.microverse.org/"> <img src="images/1.png" alt="Logo"> </a> 
+    <div class="navbar"> <img src="images/icons8-menu-.png" alt="Menu_icon"> </div>
+    </nav>
+</header>
+
+<section>
+        <h1>Charlie <br> Giorgania</h1>
+        <div class="paragraph">
+        <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+        If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+        </div>
+</section>
```

Comment:
Kindly adjust the spacing of each element in your design to match the Figma design as much as possible:

| Your Design   |   
![Screenshot from 2022-03-17 23-42-41](https://user-images.githubusercontent.com/52934128/158910663-2272fe99-b50a-414e-bc2e-e22f72a7813c.png)

| Figma Design  | 
![Screenshot from 2022-03-18 00-30-23](https://user-images.githubusercontent.com/52934128/158910615-c613539d-43af-480a-8907-c4bff96060e1.png)